.vscode/launch.json

```diff
@@ -0,0 +1,44 @@
+{
+   // Use IntelliSense to learn about possible attributes.
+   // Hover to view descriptions of existing attributes.
+   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
GitHub doesn't seem to agree that comments are allowed in json. But I assume this is not your comment but an auto-generated one?

Reply:
Yep, a generated one. Comments are not a part of JSON standard, but some IDEs allow and highlight them.